server/tsconfig.json

```diff
@@ -7,6 +7,7 @@
         ],
         "target": "es6",
         "lib": [
+            "esnext.asynciterable",
```

Comment:
I've never seen this library flag used before. What does it enable?

Reply:
It enables a new-ish feature `asynciterable` which is an iterator that has an `async` `next()` method, so you can iterate over things that return promises.